daml-foundations/daml-tools/daml-extension/syntaxes/daml12.tmLanguage.xml

```diff
@@ -343,6 +343,7 @@
 			| rec
       | template
       | with
+      | choice
```

Comment:
Why is there a sudden change in the indentation level from `template` onwards?

Reply:
space and tabs mixed up :(